src/arrays.js

```diff
@@ -1,61 +1,68 @@
 const getNthElement = (index, array) => {
-  // your code here
+  return array[index % array.length];
 };
-
 const arrayToCSVString = array => {
-  // your code here
+  return array.toString();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.map(String);
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  return strings.map(string => string.toUpperCase());
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+  return strings.map(string => string
+      .split('')
+      .reverse()
+      .join('')
+  );
 };
 
 const onlyEven = numbers => {
-  // your code here
+  return numbers.filter(number => number % 2 === 0);
 };
 
 const removeNthElement2 = (index, array) => {
-  // your code here
+  const arrayTwo = array.slice();
+  arrayTwo.splice(index, 1)
+  return arrayTwo;
 };
 
 const elementsStartingWithAVowel = strings => {
-  // your code here
+  const vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];
+  return strings.filter(string => vowels.includes(string[0]));
```

Comment:
The basic idea is here, but theres a few different ways to go about this. This has more repetition than is necessary, we could use a combination of the filter and includes array methods as you are doing but also include the `toLowerCase` string method:

```
  const vowels = ['a', 'e', 'i', 'o', 'u'];

  return strings.filter(string => vowels.includes(string[0].toLowerCase()));
```
A bit more concise and does the same job

Reply:
Its all to do with the `string[0].toLowerCase()` bit on the second line. Thats taking the first character of the string being looped over, and converting it to lower case. This means it doesnt matter if it was upper or lower case to start with, as it will always end up in lower case, meaning the array of vowels we're using to search through only needs the lower case versions.
Does that make sense?